wiki/Game_Modifiers/en.md

```diff
@@ -156,7 +156,7 @@ Do note that the usage of No Fail mod reduces the amount of possible [Performanc
 
 > Shortcut key: `E`
 
-> Less zoom. ("Zoom" meaning here means how fast the beatmap's BPM is)
+> Less zoom. ("Zoom" referring to how fast the beatmap's BPM is)
```

Comment:
* "'Zoom' referring to the beatmap's BPM"? i don't feel this needs the 'fast' modifier